### PR TITLE
Fixes #22200 : Prepare all man pages for release

### LINKED ISSUE #22200: Prepare all man pages for release
We need to make the following changes to all the manages :
1. Some of the man page headers are corrupted for ex.
list-batch-runtime-configuasadmin1Utility Slist-batch-runtime-configuration(1)

2. Change "Java EE 7" -> "Java EE 8" in footers

3. Fix all examples that show "glassfish3"

4. In all the updated man pages, make sure year is correct in footer i.e date should reflect the date on which the page content was updated/fixed

### PR DESCRIPTION
**Pull request by pranjal10**
Fixes #22200

### CODE REVIEW

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

File: appserver/admin/cli-optional/src/main/manpages/com/sun/enterprise/admin/cli/optional/backup-domain.1 [hunk 1]
```diff
@@ -79,4 +79,4 @@ EXIT STATUS
 SEE ALSO
        restore-domain(1), list-backups(1), suspend-domain(1), resume-domain(1)
 
-Java EE 7                         22 Dec 2010                 backup-domain(1)
+Java EE 8                         22 Dec 2010                 backup-domain(1)
```
Comment (arindam-bandyopadhyay) [inline appserver/admin/cli-optional/src/main/manpages/com/sun/enterprise/admin/cli/optional/backup-domain.1]: do we need to update the year? This comment is applicable for all where year is not updated

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

**Comment by pranjal10** [comment]
@glassfishrobot Run CI tests please

**Review by yaminikb — APPROVED** [review]
Looks good

**Pull request merged**